seastar_cmake.py

```diff
@@ -21,10 +21,15 @@
 
 ROOT_PATH = os.path.realpath(os.path.dirname(__file__))
 
-BUILD_PATHS = { mode: os.path.join(ROOT_PATH, 'build', mode) for mode in SUPPORTED_MODES }
-
 COOKING_BASIC_ARGS = ['./cooking.sh']
 
+def build_path(mode, build_root = 'build'):
```

Comment:
i'd suggest removing the default value of this parameter, as `build_path()` is an internal helper method only used by these two scripts, which in turn both rightly specify the `build_root` parameter explicitly.

furthermore, i think it'd be better to encode the default build root in a single place, so that we don't need to have it in both `test.py` and `configure.py` to make them work with the default settings. a global variable would suffice, like
```py
DEFAULT_BUILD_ROOT = 'build'
```
and reuse this variable when calling `parser.add_argument('--build-root', action='store', default='build'...)`.

Reply:
I've removed the default value so it must be explicitly specified.

I've ended the default build root in the seastar_cmake module and used it test.py and configure.py as you've suggested.